examples/visual-question-answering/README.md

```diff
@@ -0,0 +1,31 @@
+<!---
+Copyright 2021 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Visual Question Answering Examples
+
+This directory contains a script that showcases how to use transformers pipeline API to run visual question answering task on HPUs.
+
+## Single-HPU inference
+
+```bash
+python3 run_pipeline.py \
+    --model_name_or_path Salesforce/blip-vqa-capfilt-large \
+    --image_path "https://storage.googleapis.com/sfr-vision-language-research/BLIP/demo.jpg" \
+    --question "how many dogs are in the picture?" \
+    --use_hpu_graphs \
+    --bf16
+```
```

Comment:
when I ran this cmd, i got
The model 'BlipForQuestionAnswering' is not supported for visual-question-answering. Supported models are ['Blip2ForConditionalGeneration', 'ViltForQuestionAnswering'].
Did you see it?

Reply:
yes, I see it. but it's just warning, caused by https://github.com/huggingface/transformers/blob/main/src/transformers/models/auto/modeling_auto.py#L961

Reply:
I opened a PR to fix that: https://github.com/huggingface/transformers/pull/29499